RecoVertex/BeamSpotProducer/python/BeamSpot_cff.py

```diff
@@ -1,4 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 from RecoVertex.BeamSpotProducer.BeamSpot_cfi import *
+from RecoVertex.BeamSpotProducer.offlineBeamSpotCUDA_cfi import offlineBeamSpotCUDA
 
+offlineBeamSpotTask = cms.Task(offlineBeamSpot)
```

Comment:
Where is this BeamSpot_cff.py included?
Shouldn't this offlineBeamSpotTask get included in the [globalreco_trackingTask](https://github.com/cms-sw/cmssw/blob/master/Configuration/StandardSequences/python/Reconstruction_cff.py#L106), instead of just the `offlineBeamSpot` module?

Reply:
Indeed, that's the case in the Patatrack branch: https://github.com/cms-sw/cmssw/blob/0e7177c6c754af7e338a67692a7deb66f7005baa/Configuration/StandardSequences/python/Reconstruction_cff.py#L106-L112

I was going to add it once there is a consumer for it (OK, I missed it 😇), but if you prefer I can add it already now.

Reply:
I think that whatever we can make identical in the Patatrack branch and in the main one should be done.
Here, everything is regulated by a modifier, so adding the offlineBeamSpotTask to the globalreco_trackingTask won't affect the standard workflows.

Reply:
OK, will do.

Reply:
Done.